RHEL6_7/system/perl/check

```diff
@@ -23,6 +23,14 @@ EOM
 
 PERL_DIRS=$(perl -MConfig -e '$,=q{ }; print @Config{installarchlib,installprivlib,installvendorarch,installvendorlib}') \
     || exit_error
+#Limiting the lookup to local directories only
+for dir in ${PERL_DIRS[@]};
+do 
+    fs_type=$(stat -f -L -c %T "$dir")
+    if  echo "$fs_type" | egrep -q 'nfs|nfs4|autofs|cifs|sshfs'; then
```

Comment:
I can't find a source of all the possible values. @jmazanek , do you have such list?
For example my USB-connected disk is of type `fuseblk` which I even didn't know it exist as a filesystem type.

Reply:
No, these are just the most common remote filesystems I could recall.  Didn't consider USB disks, but I can add more

Reply:
@jmazanek: Generic solution can be done in similar way how we create list of local executables. Create list of local directories, ignoring symlinks and than check whether directory is part of the list. But it will consume some amount of time, so not sure about benefit at all.

Reply:
Yeah, the Perl module is already the second most time consuming module, so I wouldn't be adding such time consuming stuff.
BTW, I think the module is slow because of how it's written. IOW, it can be greatly optimized IMO. But that's not the subject of this PR.